themes/bootstrap5/templates/_ui/components/page-title.phtml

```diff
@@ -0,0 +1,14 @@
+<?php
+  $titleTranslated = $this->translate($this->title);
```

Comment:
We probably don't want to assume that every title should be translated -- maybe we need a variable for this. (Could default to translated = true, but there may be situations where the title is dynamic or needs to be constructed using placeholders, in which case it might be easier to translate it outside and pass it in).